### PR TITLE
fix(errors inbox): added version tracking

### PR DESCRIPTION
**Pull request by homelessbirds**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/errors-inbox/version-tracking.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    * PHP
+    * Python
+    * Ruby
+* For mobile entities, configure `appVersion`.
+* For browser entities, use `application.version`. 
+</Step>
+
+<Step>
+## Filter by version
+Once you have instrumented version tracking, your error groups will automatically capture data from each release. You can filter by each verison.
```
Comment (mjvick) [inline src/content/docs/errors-inbox/version-tracking.mdx:46]:
```suggestion
Once you have instrumented version tracking, your error groups will automatically capture data from each release. You can filter by each version.
```

File: src/content/docs/errors-inbox/version-tracking.mdx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+To filter error groups with multiple versions, use  the + operator to choose an `OR` condition.
```
Comment (mjvick) [inline src/content/docs/errors-inbox/version-tracking.mdx:48]:
```suggestion
To filter error groups with multiple versions, use the + operator to choose an `OR` condition.
```

File: src/content/docs/errors-inbox/version-tracking.mdx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+<img
+    title="version tracking errors inbox"
+    alt="A screenshot depicting how to filter by versions in errors inbox. "
+    src={errorsinboxFilterErrorsbyVersions}
+/>
+
+You can use [change tracking](/docs/change-tracking/change-tracking-view-analyze/) to monitor your deployments. If you have done this, it's important to make sure that your deployments match the same version format you use for errors inbox so you can match each deployment.
```
Comment (mjvick) [inline src/content/docs/errors-inbox/version-tracking.mdx:56]:
```suggestion
You can use [change tracking](/docs/change-tracking/change-tracking-view-analyze/) to monitor your deployments. If you have done this, it's important to make sure that your deployments match the same version format you instrumented on your entities so you can match versions from your errors to your deployments.
```

File: src/content/docs/errors-inbox/version-tracking.mdx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+metaDescription: "Errors inbox is an error tracking solution designed to give you the tools to proactively detect, triage, and resolve errors across your full application stack."
+---
+
+import errorsinboxFilterErrorsbyVersions from 'images/errors-inbox_screenshot-crop_filter-errors-by-versions.webp'
+
+Errors inbox version tracking allows developers to monitor each deployment to check for accuracy and success. Lets say you and your team an alert that says one of your entities has a spike in error rates. You identify the [source of the error](docs/tutorial-error-tracking/respond-outages/) and decide to release a canary deployment. You can use version tracking to monitor your fix as well as the previous deployments to determine if you solved the problem or if you need to go back to the drawing board. 
+
+<Steps>
+<Step>
+## Instrument version tracking on your entities.
```
Comment (mjvick) [inline src/content/docs/errors-inbox/version-tracking.mdx:12]: This section was to tell the user how the fields they will instrument are used in Errors inbox and then the 'configure fields' section below tells them how to instrument them. It is helpful for them to know what order we use the fields so they will understand  if they have `service.version` and `tags.commit `both set - we will use `service.version`.

As it is currently written, it seems repetitive and doesn't give them that key info on how we will use the fields.

File: src/content/docs/errors-inbox/version-tracking.mdx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+import errorsinboxFilterErrorsbyVersions from 'images/errors-inbox_screenshot-crop_filter-errors-by-versions.webp'
+
+Errors inbox version tracking allows developers to monitor each deployment to check for accuracy and success. Lets say you and your team an alert that says one of your entities has a spike in error rates. You identify the [source of the error](docs/tutorial-error-tracking/respond-outages/) and decide to release a canary deployment. You can use version tracking to monitor your fix as well as the previous deployments to determine if you solved the problem or if you need to go back to the drawing board. 
+
+<Steps>
+<Step>
+## Instrument version tracking on your entities. 
+
+To track errors for each deployment in APM and OpenTelemetry applications, make sure sure  the following event fields are checked:
```
Comment (mjvick) [inline src/content/docs/errors-inbox/version-tracking.mdx:14]:
```suggestion
To track errors for each deployment in APM and OpenTelemetry applications, make sure the following event fields are checked:
```

File: src/content/docs/errors-inbox/version-tracking.mdx [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+For browser applications, the event field is `application.version`.
+</Step>
+
+<Step>
+## Configure fields 
+
+To capture data from our version tracking, you must configure fields for relevant entities:
+
+* For OpenTelemetry entities, configure `service.version`.
```
Comment (mjvick) [inline src/content/docs/errors-inbox/version-tracking.mdx:31]: will each of these bullets have links to the relevant section in the docs?

File: src/content/docs/errors-inbox/version-tracking.mdx [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+    src={errorsinboxFilterErrorsbyVersions}
+/>
+
+You can use [change tracking](/docs/change-tracking/change-tracking-view-analyze/) to monitor your deployments. If you have done this, it's important to make sure that your deployments match the same version format you use for errors inbox so you can match each deployment.
+</Step>
+
+<Step>
+## Find versions by error group
+
+You can also click an error group to see each deployment that matches the first and last seen dates for your error group. You will only see versions that were live within 30 minutes after your first seen date and within 30 minutes before your last seen date.
```
Comment (mjvick) [inline src/content/docs/errors-inbox/version-tracking.mdx:62]:
```suggestion
You can also click on an error group to see each deployment that matches the first and last seen dates for your error group. You will only see versions that were live within 30 minutes after your first seen date and within 30 minutes before your last seen date.
```

File: src/content/docs/errors-inbox/version-tracking.mdx [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+You can dive deeper into your error groups by using the chart that shows each of your error occurrences grouped  by version.
```
Comment (mjvick) [inline src/content/docs/errors-inbox/version-tracking.mdx:64]:
```suggestion
You can dive deeper into your error groups by using the chart that shows each of your error occurrences grouped by version. 
```

File: src/content/docs/errors-inbox/version-tracking.mdx [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+</Step>
+
+<Step>
+## Find versions by error group
+
+You can also click an error group to see each deployment that matches the first and last seen dates for your error group. You will only see versions that were live within 30 minutes after your first seen date and within 30 minutes before your last seen date.
+
+You can dive deeper into your error groups by using the chart that shows each of your error occurrences grouped  by version. 
+
+If you have also set up deployments with change tracking, you will see deployment markers on this chart.  These makers indicate when each deployment was released and if the versions you set up on your entity match the versions on the deployments. Click the marker to navigate to those deployments.
```
Comment (mjvick) [inline src/content/docs/errors-inbox/version-tracking.mdx:66]:
```suggestion
If you have also set up deployments with change tracking, you will see deployment markers on this chart.  These markers indicate when each deployment was released. Click the marker to navigate to the deployment.
```

**Review by mjvick — COMMENTED** [review]
I added some comments/fixes. I also have a screenshot of the detail view chart now, if you would like that.

File: src/content/docs/errors-inbox/version-tracking.mdx (hunk 6 above)
Comment (homelessbirds) [reply src/content/docs/errors-inbox/version-tracking.mdx:31]: Yes!

**Comment by meda123** [comment]
This looks good @homelessbirds

**Pull request merged**